strawberry/codegen/query_codegen.py

```diff
@@ -648,17 +648,19 @@ def _unwrap_type(
         wrapper = None
 
         if isinstance(type_, StrawberryOptional):
-            type_, wrapper = self._unwrap_type(type_.of_type)
+            type_, previous_wrapper = self._unwrap_type(type_.of_type)
             wrapper = (
                 GraphQLOptional
-                if wrapper is None
-                else lambda t: GraphQLOptional(wrapper(t))  # type: ignore[misc]
+                if previous_wrapper is None
+                else lambda t: GraphQLOptional(previous_wrapper(t))  # type: ignore[misc]
             )
 
         elif isinstance(type_, StrawberryList):
-            type_, wrapper = self._unwrap_type(type_.of_type)
+            type_, previous_wrapper = self._unwrap_type(type_.of_type)
             wrapper = (
-                GraphQLList if wrapper is None else lambda t: GraphQLList(wrapper(t))
+                GraphQLList
+                if previous_wrapper is None
+                else lambda t: GraphQLList(previous_wrapper(t))
```

Comment:
I'm seeing a MyPy error on the second `wrapper = ...` assignment here. I'm guessing that's why the first one has the `#type: ignore[misc]` annotations? If so, would it be safe to add those annotations to the second one?

(ps, I'm the same person as the author, just on a different account)